### PR TITLE
[Core] Removing incorrect lumping factors

### PR DESCRIPTION
**Pull request by loumalouomega**
I had not removed these lumping factors in the previous PR. Removing as commented in our meeting.

### CODE REVIEW

File: kratos/geometries/triangle_2d_3.h [hunk 1]
```diff
@@ -360,25 +360,6 @@ template<class TPointType> class Triangle2D3
...
 
-    /**
-     * @brief Lumping factors for the calculation of the lumped mass matrix
-     * @param rResult Vector containing the lumping factors
-     * @param LumpingMethod The lumping method considered. The three methods available are:
-     *      - The row sum method
-     *      - Diagonal scaling
-     *      - Evaluation of M using a quadrature involving only the nodal points and thus automatically yielding a diagonal matrix for standard element shape function
-     */
-    Vector& LumpingFactors(
```
Comment (RiccardoRossi) [inline kratos/geometries/triangle_2d_3.h:371]: this one is ok. do not remove it as it is faster

**Comment by GuillermoCasas** [comment]
From the @KratosMultiphysics/implementation-committee, we approve these changes. The correct lumping factors are currently being calculated in the base, so eliminating these implementation solves the problem. We pass it on to the @KratosMultiphysics/technical-committee for the final evaluation.

File: kratos/geometries/triangle_3d_3.h [hunk 2]
```diff
@@ -364,24 +364,6 @@ template<class TPointType> class Triangle3D3
...
-     *      - Diagonal scaling
-     *      - Evaluation of M using a quadrature involving only the nodal points and thus automatically yielding a diagonal matrix for standard element shape function
-     */
-    Vector& LumpingFactors(
-        Vector& rResult,
-        const typename BaseType::LumpingMethods LumpingMethod = BaseType::LumpingMethods::ROW_SUM
-        )  const override
-    {
-        rResult.resize( 3, false );
-        std::fill( rResult.begin(), rResult.end(), 1.00 / 3.00 );
```
Comment (RiccardoRossi) [inline kratos/geometries/triangle_3d_3.h:381]: i woukd say this is also ok ...(and faster than the generic)

**Comment by loumalouomega** [comment]
I am doing this as you told me in person, I can revert it if needed.

**Comment by loumalouomega** [comment]
I can create a test, where I create a triangle and a tetra, and I move one node in one direction and compare the solutions.

**Comment by RiccardoRossi** [comment]
sorry, if i said it i said it wrong.

it  needs to be taken out for quads since it is not correct for skewed
quads, but for triangles it is always 1/3 independently on the shape

**Comment by loumalouomega** [comment]
OK, closing then

**Pull request closed without merging**